examples/adjoints/adjoints_ex5/adjoints_ex5.C

```diff
@@ -271,15 +310,15 @@ void set_system_parameters(HeatSystem & system,
     {
 #ifdef LIBMESH_HAVE_PETSC
       PetscDiffSolver *solver = new PetscDiffSolver(system);
-      system.time_solver->diff_solver() = std::unique_ptr<DiffSolver>(solver);
+      system.time_solver->diff_solver() = UniquePtr<DiffSolver>(solver);
 #else
-      libmesh_error_msg("This example requires libMesh to be compiled with PETSc support.");
+      libmesh_error();
```

Comment:
Likewise here - we put nice error messages in on purpose, I swear!  This definitely needs fixing before merge.

Reply:
Sorry, yeah, I borrowed some code from an older application that was using adaptive time stepping.